File: light-bulb.r
total_expenses <- 61
room1_lamps <- 10
room1_days <- 21
room1_hours <- 12
room2_lamps <- 20
room2_days <- 30
room2_hours <- 8

total_lamps <- room1_lamps + room2_lamps
total_days <- room1_days + room2_days
total_hours <- room1_hours + room2_hours

room1_expenses <- (room1_lamps/total_lamps) * (room1_days/total_days) * (room1_hours/total_hours) * total_expenses
room2_expenses <- (room2_lamps/total_lamps) * (room2_days/total_days) * (room2_hours/total_hours) * total_expenses

cat("Expenses for Room 1: ", room1_expenses, " rubles\n")
cat("Expenses for Room 2: ", room2_expenses, " rubles\n")

File: light-bulb.kt
fun main() {
    val totalExpenses = 61.0
    val room1Lamps = 10
    val room1Days = 21
    val room1Hours = 12
    val room2Lamps = 20
    val room2Days = 30
    val room2Hours = 8

    val totalLamps = room1Lamps + room2Lamps
    val totalDays = room1Days + room2Days
    val totalHours = room1Hours + room2Hours

    val room1Expenses = (room1Lamps.toDouble() / totalLamps) * (room1Days.toDouble() / totalDays) * (room1Hours.toDouble() / totalHours) * totalExpenses
    val room2Expenses = (room2Lamps.toDouble() / totalLamps) * (room2Days.toDouble() / totalDays) * (room2Hours.toDouble() / totalHours) * totalExpenses

    println("Expenses for Room 1: ${room1Expenses} rubles")
    println("Expenses for Room 2: ${room2Expenses} rubles")
}
